### DISCUSSION TITLE
Github Linguist docker is not working for windows

### DISCUSSION THREAD

**Discussion by bhuvi11** [github-linguist/linguist#5474] [General]
Hello Team,

I gave the below command 
docker run --rm -v C:\Users\bhuvaneshwari\eclipse-workspace\CLI:C:\Users\bhuvaneshwari\eclipse-workspace\CLI -w C:\Users\bhuvaneshwari\eclipse-workspace\CLI -t linguist github-linguist
docker: Error response from daemon: the working directory 'C:\Users\bhuvaneshwari\eclipse-workspace\CLI' is invalid, it needs to be an absolute path.
See 'docker run --help'.

But the same is working in linux

Is there any issue in what i have given?

Thanks in advance

**Comment by lildude** [comment]
This isn't really a Linguist issue and rather a docker usage issue so you're probably better off referring to the docker documentation.

That said, your error is happening because the path you've passed to `-w` is not a valid path on Linux. The second part of the `-v` argument you've provided is also wrong. Both of these need to be valid Linux path and they need to be the same.

With that in mind, I think something like this might work assuming you're using WSL (I don't have access to a Windows host to test this):

```console
docker run --rm -v /c/Users/bhuvaneshwari/eclipse-workspace/CLI:/CLI -w /CLI -t linguist github-linguist
```

Here the `-v` arg is saying mount Windows path `C:\Users\bhuvaneshwari\eclipse-workspace\CLI` onto `/CLI` in the container and the `-w` arg is saying to use `/CLI` in the container as the working directory.

As this isn't a Linguist bug, I'm moving this to discussions.

**Reply by lildude** [reply]
Looks like this suggestion worked from your output in https://github.com/github/linguist/issues/5477 😉

**Reply by bhuvi11** [reply]
Yeah this did work :)
